vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -125,7 +130,34 @@ class TextShadowNode final : public ShadowNodeBase {
     Super::RemoveChildAt(indexToRemove);
   }
 
+  int64_t GetReactTagAtPoint(const winrt::Point &point) {
+    if (pressableCount > 0) {
+      const auto textPointer = useBlockHitTest
+          ? TextHitTestUtils::GetPositionFromPoint(GetView().as<xaml::Controls::TextBlock>(), point)
+          : VirtualTextShadowNode::HitTest(*this, point, /* hasPressableParent = */ false);
+
+      if (textPointer != nullptr) {
+        auto inlineTag = GetTag(textPointer.Parent());
+        if (inlineTag != -1) {
+          if (auto uiManager = GetNativeUIManager(GetViewManager()->GetReactContext()).lock()) {
+            const auto node = static_cast<ShadowNodeBase *>(uiManager->getHost()->FindShadowNodeForTag(inlineTag));
+            // React Native does not support events targeted to raw text nodes.
+            // Get the parent tag instead.
+            if (!std::wcscmp(node->GetViewManager()->GetName(), L"RCTRawText")) {
+              inlineTag = node->GetParent();
+            }
+            return inlineTag;
+          }
+        }
+      }
+    }
+
+    return m_tag;
+  }
+
   TextTransform textTransform{TextTransform::Undefined};
+  int pressableCount{0};
```

Comment:
(nit) - We prefix member fields with m_
m_textTransform, m_textTransform, m_ useBlockHitTest

Reply:
@rectified95 even for intentionally public fields?

Reply:
Not sure, actually. I don't see those often here.